setup.cfg

```diff
@@ -14,3 +14,9 @@ filterwarnings =
     ignore:"@coroutine" decorator is deprecated:DeprecationWarning
     ignore:Using or importing the ABCs:DeprecationWarning
     ignore:cannot collect test class 'TestClient':pytest.PytestCollectionWarning
+
+env =
+    FALCON_ASGI_WRAP_NON_COROUTINES = Y
+    FALCON_TESTING_SESSION = Y
+    # PYTHONASYNCIODEBUG is optional
+    PYTHONASYNCIODEBUG = 1
```

Comment:
If `PYTHONASYNCIODEBUG` is optional, maybe we could resist the temptation to set it by default?
It adds horrible overhead (50x and more in some cases) to asyncio tests.

Reply:
I left it here because it is set in tox.ini. But it's optional in the sense that I use it without it and everything works.
I can comment it

Reply:
I'm fine with it either way.
But it probably does make sense to comment it out because just running `pytest` does not necessarily do other things `tox` does either. It just does what it does -- runs `pytest`.